en/documentation/index.md

```diff
@@ -71,6 +79,61 @@ The authoritative Ruby language documentation at [docs.ruby-lang.org][docs-rlo-e
 [RubyAPI.org][rubyapi-org]
 : Easily find and browse Ruby classes, modules, and methods.
 
+[ruby-doc.org][39]
+: online API documentation
+
+[DevDocs.io][40]
+: online API documentation
+
+[Ruby QuickRef][42]
+: The ruby quick reference
+
+[rubyreferences][43]
+    : A full language reference + detailed language changelog. Fantastic.
```

Comment:
Extra spacing:

```suggestion
: A full language reference + detailed language changelog. Fantastic.
```

While I agree it's a fantastic work, it feels a bit odd to add `Fantastic.` in its description?